app/Observers/PostObserver.php

```diff
@@ -71,6 +73,42 @@ public function creating(Post $post)
         }
     }
 
+    /**
+     * @param Post $post
+     */
+    public function created(Post $post)
+    {
+        $log = new EntityLog();
+        $log->entity_id = $post->entity->id;
+        $log->created_by = auth()->user()->id;
+        $log->entity_note_id = $post->id;
```

Comment:
Please replace entity_note_id with post_id, to reduce the work needed in the future, thanks :)